Utilities/QC/QCMerger/src/MergerDevice.cxx

```diff
@@ -36,8 +36,8 @@ MergerDevice::MergerDevice(unique_ptr<Merger> merger, string mergerId, int numIo
   : mMerger(move(merger)), ddsCustomCmd(new CCustomCmd(mService))
 {
   this->SetTransport("zeromq");
-  this->SetProperty(Id, mergerId);
-  this->SetProperty(NumIoThreads, numIoThreads);
+  this->SetId(mergerId);
+  this->SetNumIoThreads(numIoThreads);
```

Comment:
@rbx, @Barthelemy can this probably be deprecated? Setting the number of IO threads is probably a remnant from the early times.

Reply:
@rbx, then I suggest you delete to calls to `Get/SetNumIoThreads` in the course of these changes

Reply:
done.